maintenance_evolution.Rmd

```diff
@@ -201,6 +201,19 @@ You don't have to change the tests of deprecated functions until they are made d
 
 Once a function is made defunct, its tests are simply removed.
 
+## Renaming packages {#renaming-packages}
+
+Renaming a package that is in very early development is fine.
+It can be the opportunity, before review, to comply with our [naming advice](#naming-your-package).
+
+Now, renaming a package that has been widely adopted and/or released on CRAN is problematic.
+CRAN has a [policy](https://cran.r-project.org/web/packages/policies.html) stating that Package names on CRAN are persistent and in general it is not permitted to change a package’s name.
+The package under its old name might be a dependency of packages, scripts, and feature in documentation and blog posts, etc.
+
+Creating a newer package that is a rewrite with a better interface, like [httr2 which is the second generation of httr](https://www.tidyverse.org/blog/2023/11/httr2-1-0-0/) ; or creating editions of a package like for [testthat](https://testthat.r-lib.org/articles/third-edition.html), are better strategies.
```

Comment:
```suggestion
Instead of renaming, it is better to create a newer package that is a rewrite with a better interface, like [httr2 which is the second generation of httr](https://www.tidyverse.org/blog/2023/11/httr2-1-0-0/) ; or to create editions of a package like for [testthat](https://testthat.r-lib.org/articles/third-edition.html).
```